cmd/mistryd/main.go

```diff
@@ -114,22 +139,28 @@ func main() {
 
 // SetUp accepts a Config and performs necessary initialization tasks.
 func SetUp(cfg *Config) error {
-	err := utils.PathIsDir(cfg.ProjectsPath)
+	err := PruneZombieBuilds(cfg)
 	if err != nil {
 		return err
 	}
 
-	err = utils.PathIsDir(cfg.BuildPath)
+	return nil
+}
+
+func parseAndValidateConfig(c *cli.Context) (*Config, error) {
```

Comment:
could just be "parseConfig"

Reply:
the function `ParseConfig` already exists and parses the JSON, this one calls it and validates results

maybe it should just be 1 function

Reply:
n/m I'm renaming to `parseConfigFromCli`. it knows how to turn CLI flags into a proper config. the `ParseConfig` function remains, but knows nothing about the CLI flags